Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 Flask = "*"
-attrs = "==19.1.0"
+attrs = "==19.2.0"
 jsonpointer = "*"
 eq-translations = {editable = true,git = "https://github.com/ONSDigital/eq-translations.git",ref = "v2.0.0"}
```

Comment:
Can't quite remember why we never upgraded this?

Reply:
If we don't upgrade it, linting fails with an invalid kwarg argument error for the equality `eq` kwarg

Reply:
I meant the eQ translations package.

Reply:
We should remove this and see if it works as it's not a direct requirement. The reason for it being there was probably to fix a transitive dependency issue that has been resolved.

Reply:
I'll give it a try